Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -37,7 +37,7 @@ Set-Alias -Scope Global Reset-MinimalTemplateFromSamples Initialize-MinimalTempl
 Set-DeploymentSettingsFiles @(
     "$(Get-RepositoryResolvedPath 'Application\EdFi.Ods.WebApi.NetCore')\appsettings.json",
     "$(Get-RepositoryResolvedPath 'Application\EdFi.Ods.WebApi.NetCore')\appsettings.development.json",
-    "$(Get-RepositoryResolvedPath 'Application\EdFi.Ods.WebApi.NetCore')\appsettings.user.json"
+    "$env:APPDATA\Microsoft\UserSecrets\f1506d66-289c-44cb-a2e2-80411cc690ec\secrets.json"
```

Comment:
```suggestion
   Join-Path ([Environment]::GetEnvironmentVariable("AppData")) 'Microsoft/UserSecrets/f1506d66-289c-44cb-a2e2-80411cc690ec/secrets.json'
```

Reply:
The powershell should only be accessing this file through the CLI.  If you hard code paths you risk breaking if they change something about the path behind the CLI.

Reply:
I think that makes sense but is going to require additional effort to parse the falttented settings from the `dotnet user-secrets list` output.

Reply:
scap my idea and go with @blmeyers idea

Reply:
Implemented as per ben suggestion now. Thank you !